raiden/network/transport/matrix/client.py

```diff
@@ -241,6 +242,12 @@ def hangup(self, room_id: RoomID, call_id: str) -> None:
         message = {"type": RTCMessageType.HANGUP.value, "call_id": call_id}
         self._send_signalling(room_id, message)
 
+    def send_to_device_message(self, user_ids: Iterable[str], text: str) -> None:
+        # TODO: send just to active user id
+        data = {user_id: {"*": text} for user_id in user_ids}
+
+        self.send_to_device(event_type="m.text", messages=data)
```

Comment:
Here.. `event_type` must be `m.room.message` (even though it isn't a room, this is what the matrix spec call this kind of event), and `content` must be on the form `{ "msgtype": "m.text", "body": str }`